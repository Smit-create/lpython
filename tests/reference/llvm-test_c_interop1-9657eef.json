{
    "basename": "llvm-test_c_interop1-9657eef",
    "cmd": "lpython --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/errors/test_c_interop1.py",
    "infile_hash": "750a2bb08545b8c11e44585855545060693c59a71df5989ac5371720",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-test_c_interop1-9657eef.stdout",
    "stdout_hash": "85812598f4361e1c59c05aed149ed350a440886bb5961ab0bffcb4e0",
    "stderr": "llvm-test_c_interop1-9657eef.stderr",
    "stderr_hash": "de1cdb0fb09880b7f55321d292ae4a3a4bac329966ccfcebc36d7481",
    "returncode": 3
}