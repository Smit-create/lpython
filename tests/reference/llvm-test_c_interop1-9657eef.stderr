code generation error: asr_to_llvm: module failed verification. Error:
Stored value type does not match pointer operand type!
  store %array* %y, %array.0** %yptr1, align 8
 %array.0*
